jbmc/src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -494,20 +493,23 @@ optionalt<codet> java_bytecode_instrumentt::instrument_expr(const exprt &expr)
     {
       // this corresponds to a throw and so we check that
       // we don't throw null
-      result.add(check_null_dereference(expr.op0(), expr.source_location()));
+      result.add(check_null_dereference(
+        to_unary_expr(expr).op(), expr.source_location()));
```

Comment:
We have `side_effect_expr_throwt`

Reply:
yes, but that doesn't have operands -- will make that a separate PR.